parser.go

```diff
@@ -172,15 +167,19 @@ func NewRDJSONLParser() *RDJSONLParser {
 	return &RDJSONLParser{}
 }
 
-func (p *RDJSONLParser) Parse(r io.Reader) ([]*CheckResult, error) {
-	var results []*CheckResult
+func (p *RDJSONLParser) Parse(r io.Reader) ([]*rdf.Diagnostic, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method RDJSONLParser.Parse should have comment or be unexported